tests/backstop_data/engine_scripts/puppet/utility.js

```diff
@@ -30,25 +105,91 @@ module.exports = class CrmPage {
 
   /**
    * Waits for all the loading placeholders to vanish
+   *
+   * @param {String} parentSelector
+   *  - for contextual loading checks
    */
-  async waitForLoadingComplete () {
-    await this.engine.waitFor(() => {
-      const allLoadingElements = document.querySelectorAll('div[class*="civicase__loading-placeholder"]');
+  async waitForLoadingComplete (parentSelector) {
+    const timeout = 100000;
```

Comment:
Not sure, what this timeout is used for. 
And should not we reuse the timeout set in `backstop.tpl.json`?
```
  "engineOptions": {
    "waitTimeout": 120000
  },
```

Reply:
The whole function waits for loading placeholders to disappear before proceeding. The `engineOptions` `waitTimeout` defined in the `backstop.tpl.json` is specific to the puppeteer default timeout. There's no reliable way to get this timeout and use it inside of the `utility.js` script, hence this custom one is used. 

Maybe I could bump it to `120000` here as well?

Reply:
okay got it. lets keep it, np.